schema/dsl/parse_test.go

```diff
@@ -35,6 +38,62 @@ func TestParseSchemaSchema(t *testing.T) {
 	})
 }
 
+type yamlFixture struct {
```

Comment:
[Sadness documented elsewhere](https://github.com/ipld/ipld/pull/142#issuecomment-947677223).  I'll say no more here :) but do consider this something I'd like to replace with not-yaml sooner than later.

Not hanging up this PR on it, though.  Too much other good stuff that worth keeping moving.

Reply:
Certainly. This is copy-pasted from Rod's go-ipld-prime, so it's not like I spent more than a minute on it.